packages/dashmate/configs/getConfigFileMigrationsFactory.js

```diff
@@ -113,6 +113,17 @@ function getConfigFileMigrationsFactory(homeDir, defaultConfigs) {
           });
         return configFile;
       },
+      '0.24.22': (configFile) => {
+        Object.entries(configFile.configs)
+          .forEach(([, options]) => {
+            if (options.core.masternode.enable) {
+              options.platform.drive.tenderdash.mode = 'validator';
```

Comment:
Aren't only evonodes, and not any masternode are validators?

Reply:
Yes, but both have platform sections, it just disabled. We cant leave it undefined, because then node config will be invalid (compared to schema)